test_retinanet onnx

**Describe the bug**
Pytorch version: 2.0.1+cu117

```
[2023-05-25T02:36:28.192Z] ======================================================================
[2023-05-25T02:36:28.192Z] FAIL: test_onnx_3 (tests.test_retinanet.TestRetinaNet)
[2023-05-25T02:36:28.192Z] ----------------------------------------------------------------------
[2023-05-25T02:36:28.192Z] Traceback (most recent call last):
[2023-05-25T02:36:28.192Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 620, in standalone_func
[2023-05-25T02:36:28.192Z]     return func(*(a + p.args), **p.kwargs, **kw)
[2023-05-25T02:36:28.192Z]   File "/home/jenkins/agent/workspace/MONAI-postmerge/monai_github/tests/test_retinanet.py", line 186, in test_onnx
[2023-05-25T02:36:28.192Z]     test_onnx_save(backbone, data, rtol=2e-3)
[2023-05-25T02:36:28.192Z]   File "/home/jenkins/agent/workspace/MONAI-postmerge/monai_github/tests/utils.py", line 760, in test_onnx_save
[2023-05-25T02:36:28.192Z]     convert_to_onnx(
[2023-05-25T02:36:28.192Z]   File "/home/jenkins/agent/workspace/MONAI-postmerge/monai_github/monai/networks/utils.py", line 700, in convert_to_onnx
[2023-05-25T02:36:28.192Z]     assert_fn(r1.cpu(), convert_to_tensor(r2, dtype=r1.dtype), rtol=rtol, atol=atol)  # type: ignore
[2023-05-25T02:36:28.192Z]   File "/opt/conda/lib/python3.8/site-packages/torch/testing/_comparison.py", line 1511, in assert_close
[2023-05-25T02:36:28.192Z]     raise error_metas[0].to_error(msg)
[2023-05-25T02:36:28.192Z] AssertionError: Tensor-likes are not close!
[2023-05-25T02:36:28.192Z] 
[2023-05-25T02:36:28.192Z] Mismatched elements: 1 / 9 (11.1%)
[2023-05-25T02:36:28.192Z] Greatest absolute difference: 4.26173210144043e-06 at index (2, 1) (up to 0.0 allowed)
[2023-05-25T02:36:28.192Z] Greatest relative difference: 0.01431574732205426 at index (2, 1) (up to 0.002 allowed)
[2023-05-25T02:36:28.192Z] 
[2023-05-25T02:36:28.192Z] ----------------------------------------------------------------------
```

jenkins/MONAI-postmerge/detail/MONAI-postmerge/1399/pipeline/


still an issue
```
[2023-06-14T00:11:03.039Z] ======================================================================
[2023-06-14T00:11:03.039Z] FAIL: test_onnx_3 (tests.test_retinanet.TestRetinaNet)
[2023-06-14T00:11:03.039Z] ----------------------------------------------------------------------
[2023-06-14T00:11:03.039Z] Traceback (most recent call last):
[2023-06-14T00:11:03.039Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 620, in standalone_func
[2023-06-14T00:11:03.039Z]     return func(*(a + p.args), **p.kwargs, **kw)
[2023-06-14T00:11:03.039Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/tests/test_retinanet.py", line 186, in test_onnx
[2023-06-14T00:11:03.039Z]     test_onnx_save(backbone, data, rtol=2e-3)
[2023-06-14T00:11:03.039Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/tests/utils.py", line 760, in test_onnx_save
[2023-06-14T00:11:03.039Z]     convert_to_onnx(
[2023-06-14T00:11:03.039Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/monai/networks/utils.py", line 700, in convert_to_onnx
[2023-06-14T00:11:03.039Z]     assert_fn(r1.cpu(), convert_to_tensor(r2, dtype=r1.dtype), rtol=rtol, atol=atol)  # type: ignore
[2023-06-14T00:11:03.039Z]   File "/opt/conda/lib/python3.8/site-packages/torch/testing/_comparison.py", line 1344, in assert_close
[2023-06-14T00:11:03.039Z]     assert_equal(
[2023-06-14T00:11:03.039Z]   File "/opt/conda/lib/python3.8/site-packages/torch/testing/_comparison.py", line 1095, in assert_equal
[2023-06-14T00:11:03.039Z]     raise error_metas[0].to_error(msg)
[2023-06-14T00:11:03.039Z] AssertionError: Tensor-likes are not close!
[2023-06-14T00:11:03.039Z] 
[2023-06-14T00:11:03.039Z] Mismatched elements: 1 / 9 (11.1%)
[2023-06-14T00:11:03.039Z] Greatest absolute difference: 1.430511474609375e-06 at index (1, 0) (up to 0.0 allowed)
[2023-06-14T00:11:03.039Z] Greatest relative difference: 0.004467609828741623 at index (1, 0) (up to 0.002 allowed)
[2023-06-14T00:11:03.039Z] 
[2023-06-14T00:11:03.039Z] ----------------------------------------------------------------------
```

jenkins/pipelines/Monai-pytorch-versions/runs/949

--- a/tests/test_retinanet.py
+++ b/tests/test_retinanet.py
@@ -183,7 +183,7 @@ class TestRetinaNet(unittest.TestCase):
         data = torch.randn(input_shape)
         backbone = model(**input_param)
         if idx == 0:
-            test_onnx_save(backbone, data, rtol=2e-3)
+            test_onnx_save(backbone, data, rtol=2e-2, atol=1e-5)
             return
         feature_extractor = resnet_fpn_feature_extractor(
             backbone=backbone,
@@ -193,7 +193,7 @@ class TestRetinaNet(unittest.TestCase):
             returned_layers=[1, 2],
         )
         if idx == 1:
-            test_onnx_save(feature_extractor, data, rtol=2e-3)
+            test_onnx_save(feature_extractor, data, rtol=2e-2, atol=1e-5)
             return
         net = RetinaNet(
             spatial_dims=input_param["spatial_dims"],
@@ -203,7 +203,7 @@ class TestRetinaNet(unittest.TestCase):
             size_divisible=32,
         )
         if idx == 2:
-            test_onnx_save(net, data, rtol=2e-3)
+            test_onnx_save(net, data, rtol=2e-2, atol=1e-5)
 
 
 if __name__ == "__main__":

